infra/modules/aws-psoxy-lambda/main.tf

```diff
@@ -30,20 +30,6 @@ resource "aws_cloudwatch_log_group" "lambda-log" {
   retention_in_days = var.log_retention_in_days
 }
 
-resource "aws_lambda_function_url" "lambda_url" {
```

Comment:
Ah, cool. I've seen an error meanwhile helping David on proxy and after checking this, this is the root cause, not the thing I've included in my ticket -> https://app.asana.com/0/1202815052819390/1202851274640352/f